README.md

```diff
@@ -21,6 +21,7 @@ Drop by our slack channel (#playback) on the [Video.js slack](http://slack.video
 - [Getting Started](#getting-started)
 - [Protected Content](#protected-content)
 - [Captions](#captions)
+- [Multiple Language Labels](#multiple-language-labels)
```

Comment:
I think `Multi-Language Labels` is better here, as "Multiple Language Labels" could be understood as "Multiple (Language Labels)" when what we mean is "(Multiple Language) Labels", if you get what I mean.